client/trino-client/src/test/java/io/trino/client/TestSSL.java

```diff
@@ -67,7 +67,7 @@ public void setUp()
     {
         Bootstrap app = new Bootstrap(
                 new TestingNodeModule(),
-                new TestingHttpServerModule(),
+                new TestingHttpServerModule("test-ssl"),
```

Comment:
fix author of `Bump io.airlift:bom from 380 to 381` commit. it's 99% @wendigo 's, not @dependabot 's